src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -891,6 +891,21 @@ def create(self, vol_name: str, subvol_name: str, snap_name: str, group_name='')
             )
         return f'Subvolume snapshot {snap_name} created successfully'
 
+    def delete(self, vol_name: str, subvol_name: str, snap_name: str, group_name='', force=True):
+        params = {'vol_name': vol_name, 'sub_name': subvol_name, 'snap_name': snap_name}
+        if group_name:
+            params['group_name'] = group_name
+        force = str_to_bool(force)
+        if force:
+            params['force'] = 'True'
```

Comment:
`force=True` by default? Is there any particular reason for that?

Reply:
Nope, maybe I can introduce another checkbox in the UI for forcing it but in this force is harmless for normal deletion.. 
`Using the --force flag allows the command to succeed that would otherwise fail if the snapshot did not exist.`

So rather than going for an extra checkbox I just went ahead and made it True always

Reply:
ah but i should be using `params['force'] = force.

Reply:
addressed!

Reply:
Thanks for the info!